docs/resources/faraway_replica.md

```diff
@@ -228,8 +252,17 @@ Read-Only:
 <a id="nestedatt--cluster_architecture"></a>
 ### Nested Schema for `cluster_architecture`
 
+Required:
+
+- `id` (String) Cluster architecture ID.
+- `nodes` (Number) Node count.
+
 Read-Only:
 
-- `id` (String) Cluster architecture ID. For example, "single" or "ha".For Extreme High Availability clusters, please use the [biganimal_pgd](https://registry.terraform.io/providers/EnterpriseDB/biganimal/latest/docs/resources/pgd) resource.
 - `name` (String) Name.
-- `nodes` (Number) Node count.
+
+## Import
+
+Import is supported using the following syntax:
+
+{{codefile "shell" "/Users/amol.vyavahare/Documents/projects/terraform-provider-biganimal/examples/resources/biganimal_faraway_replica/import.sh"}}
```

Comment:
we need to change https://github.com/EnterpriseDB/terraform-provider-biganimal/blob/a60af1e2ae12c52325c00d196c3a25b49dcf7dac/templates/resources/faraway_replica.md.tmpl#L25 to `{{ codefile "shell" .ImportFile }}` and run make docs again

Reply:
Done